frontend/src/pages/projectView/ProjectView.tsx

```diff
@@ -1,18 +1,41 @@
-import { useParams } from 'react-router-dom';
 import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import allTasks from './tasks';
 import projects from './projects';
 import thumbnail from '../../assets/images/Rectangle 3025.svg';
 import './projectView.css';
 import ProjectListView from '../../components/projectListView/ProjectListView';
-import { Task } from '../../types/Task';
-import { Project } from '../../types/Project';
+import { ProjectDetails } from '../../IndexForImport';
+import { useTaskStore, initialValue } from '../../store/task.store';
+
+export type Task = {
+  id: number | string;
+  description: string;
+  status: string;
+  due_date: string;
+  assignee: string;
+  completed: boolean;
+  priority: string;
+  project_id: number | undefined;
+  user_id: number | string;
+};
+
+type Project = {
+  id: number;
+  title: string;
+  description: string;
+  thumbnail_link: string;
+  date_of_creation: string;
+  amount_of_tasks: number;
+  user_id: number;
```

Comment:
Kristina has already made these objects. You probably need to merge develop into your branch.

Also you guys should communicate so you dont step on eachothers toes :)

Reply:
True. will merge with dev